### ISSUE TITLE
Output file should be transferred via Blob, not email 

### ISSUE THREAD

**Issue by nick-gibb** [nick-gibb/points_to_polys#2]
Hey Scott V., 

Currently we are outputting a data file each week with an identical filename (`fluwatchers.csv` or something like that). 

We are emailing this to you (and Scott Thompson will email you today's dataset). But I'm assuming you prefer to fetch if from Blob. I have a container already set up there. I think we need to get you a SAS token for it. 

I'll leave this ticket open until we resolve this.

**Comment by svmillin** [comment]
Yes, preference is to retrieve the data from the blob. For the files that we currently pull, no SAS token is required.
Sidenote: We seem to have issues fetching from File Share - not sure why.
Send me the link via email and we can test this out today. Thanks!

**Comment by srt749** [comment]
Tested and working, the latest output file is now in the fluwatchers blob. 

I made a branch "srt" with code (in map_fsa_hr.py) that will upload the output file as a blob to a container named "fluwatchers" in a storage account identified by a command line argument. User must provide a key in the form of a Connection String in the same way as the other arguments, with -k "connection string here", quotes not necessary.
